OrbitService/OrbitService.cpp

```diff
@@ -5,17 +5,21 @@
 #include "Capture.h"
 #include "ConnectionManager.h"
 #include "Core.h"
+#include "CoreApp.h"
 #include "TcpServer.h"
 #include "TimerManager.h"
 
 OrbitService::OrbitService() {
   // TODO: these should be a private fields.
   GTimerManager = std::make_unique<TimerManager>();
-  GTcpServer = std::make_unique<TcpServer>();
+  GTcpServer = std::make_shared<TcpServer>();
   Capture::Init();
 
   GTcpServer->Start(Capture::GCapturePort);
   ConnectionManager::Get().InitAsService();
+
+  GCoreApp = std::make_shared<CoreApp>();
```

Comment:
How did this work before this PR? It seems like GCoreApp was always `nullptr` in OrbitService?

Reply:
this is expected - OrbitService does not use GCoreApp.

Reply:
OrbitService now uses GCoreApp to have access to managers.  We should probably centralize all managers in GCoreApp to control initialization order and life cycle.